### ISSUE TITLE
Adout train and val

### ISSUE THREAD

**Issue by Axwell360** [Shaosifan/FIANet#10]
why you are:
def main(args):
    dataset, num_classes = get_dataset("train",
                                       get_transform(args=args),
                                       args=args)

    dataset_test, _ = get_dataset("test",
                                  get_transform(args=args),
                                  args=args)
not
def main(args):
    dataset, num_classes = get_dataset("train",
                                       get_transform(args=args),
                                       args=args)

    dataset_test, _ = get_dataset("val",
                                  get_transform(args=args),
                                  args=args)
Generally speaking, shouldn't the test set be larger than the validation set?

**Comment by Shaosifan** [comment]
> why you are: def main(args): dataset, num_classes = get_dataset("train", get_transform(args=args), args=args)
> 
> ```
> dataset_test, _ = get_dataset("test",
>                               get_transform(args=args),
>                               args=args)
> ```
> 
> not def main(args): dataset, num_classes = get_dataset("train", get_transform(args=args), args=args)
> 
> ```
> dataset_test, _ = get_dataset("val",
>                               get_transform(args=args),
>                               args=args)
> ```
> 
> Generally speaking, shouldn't the test set be larger than the validation set?

Hi,  I am sorry that it should be "val" instead of "test" in the train.py. This was a mistake that occurred while reorganizing the code.
